scripts/street_sign_recognizer.py

```diff
@@ -17,21 +17,40 @@ def __init__(self):
         """ Initialize the street sign reocgnizer """
         rospy.init_node('street_sign_recognizer')
         self.cv_image = None                        # the latest image from the camera
+        self.hsv_image = None
         self.bridge = CvBridge()                    # used to convert ROS messages to OpenCV
         cv2.namedWindow('video_window')
         rospy.Subscriber("/camera/image_raw", Image, self.process_image)
+        cv2.namedWindow('threshold_image')
+        self.hsv_lb = np.array([0, 0, 0]) # hsv lower bound
+        cv2.createTrackbar('H lb', 'threshold_image', 0, 255, self.set_h_lb)
+        cv2.createTrackbar('S lb', 'threshold_image', 0, 255, self.set_s_lb)
+        cv2.createTrackbar('V lb', 'threshold_image', 0, 255, self.set_v_lb)
+        self.hsv_ub = np.array([255, 255, 255]) # hsv upper bound
+        cv2.createTrackbar('H ub', 'threshold_image', 0, 255, self.set_h_ub)
+        cv2.createTrackbar('S ub', 'threshold_image', 0, 255, self.set_s_ub)
+        cv2.createTrackbar('V ub', 'threshold_image', 0, 255, self.set_v_ub)
+
 
     def process_image(self, msg):
         """ Process image messages from ROS and stash them in an attribute
             called cv_image for subsequent processing """
         self.cv_image = self.bridge.imgmsg_to_cv2(msg, desired_encoding="bgr8")
 
+        self.hsv_image = cv2.cvtColor(self.cv_image, cv2.COLOR_BGR2HSV)
+        self.binary_image = cv2.inRange(self.hsv_image, 
```

Comment:
It looks like you used sliders to find threshold values, which is nice, but then you hard coded a set of values here without commenting out the slider code?